src/Text/Atom/Feed/Import.hs

```diff
@@ -71,15 +71,17 @@ pQLeaf :: Name -> [XML.Element] -> Maybe Text
 pQLeaf x es = (T.concat . elementText) `fmap` pQNode x es
 
 pAttr :: Text -> XML.Element -> Maybe Text
-pAttr x e = (`attributeText` e) =<< fst <$> find sameAttr (elementAttributes e)
-  where
-    ax = atomName x
-    sameAttr (k, _) = k == ax || (isNothing (nameNamespace k) && nameLocalName k == x)
+pAttr x e = (`attributeText` e) =<< find (sameAtomAttr x) (map fst $ elementAttributes e)
 
 pAttrs :: Text -> XML.Element -> [Text]
 pAttrs x e = [t | ContentText t <- cnts]
   where
-    cnts = concat [v | (k, v) <- elementAttributes e, k == atomName x]
+    cnts = concat [v | (k, v) <- elementAttributes e, sameAtomAttr x k]
```

Comment:
Hey @koterpillar thanks for the PR and the test case for this issue! I think all of this looks fine, but I was wondering if you found something in the XML spec that states that `pAttrs` was the right one to change? You previously said you thought it was that one or `atomAttr`.

Reply:
Sorry, I didn't dig into the spec. I think this fix isn't foolproof as it doesn't take into account the namespace mapping of the _element_. However, it works for the XML produced by the library itself, as well as parsed from external sources.